h/services/user.py

```diff
@@ -95,9 +95,10 @@ def fetch_all(self, userids):
 
         cache_keys = {}
         for userid in userids:
+            val = split_user(userid)
```

Comment:
This raises `InvalidUserId` which is not a child of `ValueError` so this wasn't the indended target. So :+1: